web/src/engine/osk/src/lengthStyle.ts

```diff
@@ -61,12 +61,15 @@ export class ParsedLengthStyle implements LengthStyle {
   }
 
   private static parseLengthStyle(spec: string): LengthStyle {
+    if(spec == '') {
+      return { val: 1, absolute: false };
+    }
+
     const val = parseFloat(spec);
 
     if(isNaN(val)) {
       // Cannot parse.
       console.error("Could not properly parse specified length style info: '" + spec + "'.");
```

Comment:
Given val is NaN here, what should we be returning?

```suggestion
    if(isNaN(val)) {
      // Cannot parse.
      console.error("Could not properly parse specified length style info: '" + spec + "'.");
      return { val: 1, absolute: false };
```

Reply:
I actually did this exact suggestion at first, but figured that if we ever run into a spot with an actual garbage string in production, having a clear error would be wise.  I don't see any Sentry events for the error string, and that's a bit concerning.